.ci-build.yml

```diff
@@ -77,6 +40,7 @@ requires:
     - sudo
     - texinfo
     - which
+    - zip
```

Comment:
This `zip` entry should be above the `# base-devel` comment above as it is a special dependency, not part of the *base-devel* group (which uses `bsdtar` for zip support).

I'm adding this to the AUR package for *xiphos(-git)* as well.

Reply:
Moved.